DatadogSessionReplay/Sources/Models/SRDataModels.swift

```diff
@@ -270,8 +270,8 @@ public struct SRShapeStyle: Codable, Hashable {
         opacity: Double? = nil
     ) {
         self.backgroundColor = backgroundColor
-        self.cornerRadius = cornerRadius
-        self.opacity = opacity
+        self.cornerRadius = cornerRadius.flatMap { $0.isNaN ? nil : $0 }
+        self.opacity = opacity.flatMap { $0.isNaN ? nil : $0 }
```

Comment:
🤦🏽 `SRDataModels` is generated from a JSON schema. I will move this logic to a factory method in an extension.